ee/vellum_ee/workflows/display/nodes/vellum/base_node.py

```diff
@@ -40,6 +49,20 @@ def serialize(self, display_context: WorkflowDisplayContext, **kwargs: Any) -> J
                 }
             )
 
+        if has_wrapped_node(node):
+            wrapped_node = get_wrapped_node(node)
+            display_class = get_node_display_class(BaseNodeDisplay, wrapped_node)
```

Comment:
[blocking] I'd be sure to rebase - we made some changes to these utilities last night